Changelog.md

```diff
@@ -4,6 +4,9 @@
 
 ## Gazebo 9.16.0 (2020-11-24)
 
+1. Fix support for camera with single channel floating point image format
```

Comment:
just realiezd I added the changelog entry in the wrong place. It should be in a new `Gazebo 9.xx.xx` section. cc @scpeters, let's remember to move this entry when making a new gazebo 9 release.

Reply:
I noticed it right after I merged and fixed with a direct commit to gazebo9: https://github.com/osrf/gazebo/commit/d94e2e0b6754a929c11dec4dff2131cfd291478e